src/shared/containers/MapContainer.js

```diff
@@ -0,0 +1,58 @@
+import React, {PropTypes, Component} from 'react';
+import { connect } from 'react-redux';
+
+import { getMapMarkers } from '../actions/map';
+import MapBlock from '../components/MapBlock.js';
+
+
+class MapContainer extends Component {
```

Comment:
This is a component. Container should only wire component and reducer and state mapping. Please extract this  into src/shared/compoents/...